src/DOMUpdates.js

```diff
@@ -49,9 +42,50 @@ const domUpdates = {
         element.classList.add('hidden');
       }
     })
-  }
+  },
+
+  updateWelcomeMessage(customer = {name: '- uh oh. Looks like we had an error'}) {
+    const welcomeMessage = document.querySelector('#welcome-message');
+    welcomeMessage.innerHTML = `Welcome ${customer.name}`;
+  },
+
+  displayCustomerExpenditures(customer = {totalExpenditures: 0}) {
+    const customerExpenditure = document.querySelector('#customer-expenditure');
+    customerExpenditure.innerHTML = `Your total for all bookings is ${customer.totalExpenditures}`
+  },
 
+  populateCustomerBookings(bookedRooms = []) {
+    const customerBookings = document.querySelector('#customer-bookings');
+    bookedRooms.forEach(bookedRoom => {
+      customerBookings.innerHTML += `
+      <section role="figure">
+        <p role="heading">On ${bookedRoom.dateBooked}:</p>
+        <p>Room ${bookedRoom.number}</p>
+        <p>${bookedRoom.roomType}</p>
+      </section>
```

Comment:
I'll flesh this out later on, was more concerned with just getting the correct info to show up